### PR TITLE
feat(artifact_proxy): support Flutter 3.7.10

### PR DESCRIPTION
**Pull request by felangel**
## Description

- feat(artifact_proxy): support Flutter 3.7.10

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by eseidel** [comment]
I figured we'd skip 3.7.9 and just go to 3.7.10?

**Comment by eseidel** [comment]
See also https://github.com/shorebirdtech/shorebird/pull/245

File: packages/artifact_proxy/lib/config.dart [hunk 1]
```diff
@@ -3,10 +3,41 @@ import 'package:artifact_proxy/artifact_proxy.dart';
...
 const config = ProxyConfig(
   engineMappings: {
+    ...flutter_3_7_9,
     ...flutter_3_7_8,
   },
 );
 
+/// Flutter 3.7.9
+const flutter_3_7_9 = {
+  '2b9e2639de0a81e55b4640465facc448d5e9c269': EngineMapping(
```
Comment (eseidel) [inline packages/artifact_proxy/lib/config.dart]: This revision is still 3.7.8, just with support for more platforms I think?

Comment (felangel) [reply packages/artifact_proxy/lib/config.dart]: I thought it was 3.7.9 based on the git log

**Review by eseidel — APPROVED** [review]
lgtm

**Pull request merged**